server/controllers/finance/cashboxes/index.js

```diff
@@ -54,6 +54,18 @@ function list(req, res, next) {
   filters.equals('project_id');
   filters.equals('is_auxiliary');
 
+  if (req.query.detailed === '1') {
```

Comment:
Does just `filters.equals('user_id')` not work for both cases?

Reply:
I used an alternative structure just because in the first case we use the table cash_box_account_currency and in the second it's cash_box